### ISSUE TITLE
Add or options

### ISSUE THREAD

**Issue by sospedra** [sospedra/mayre#3]
```js
<Mayre
  of={ComponentA}
  or={ComponentB}
  when={condition}
  with={{}}
  orWidth={{}}
/>
```

Hence, we can support not only conditional from 0 to 1 but from 1 to 2

**Comment by Schniz** [comment]
Sounds like an `Either`, not a `Maybe`! 😜

**Comment by jaycee425** [comment]
seems nice :D is it jsut an idea for now or is on progress?

**Comment by sospedra** [comment]
Or `this or maybe that` :P
It's a WIP! I'll finish it this weekend probably @jaycee425

**Comment by jaycee425** [comment]
Awsome! thanks for the work really useful!

**Comment by sospedra** [comment]
@jaycee425 Either feature si released under `2.2.0` ;)
